include/Field.h

```diff
@@ -23,6 +23,9 @@ SET_GLOBAL( FieldIdx_t Idx_MomX,          Idx_Undefined );
 SET_GLOBAL( FieldIdx_t Idx_MomY,          Idx_Undefined );
 SET_GLOBAL( FieldIdx_t Idx_MomZ,          Idx_Undefined );
 SET_GLOBAL( FieldIdx_t Idx_Engy,          Idx_Undefined );
+#if ( EOS == EOS_NUCLEAR )
+SET_GLOBAL( FieldIdx_t Idx_Ye,            Idx_Undefined );
```

Comment:
We could always use the Eos solver to get the entropy but would be useful (though cost memory) to add another filed for entropy.

Reply:
I'll add entropy in a separate PR.